packages/gatsby/src/utils/static-query-utils.ts

```diff
@@ -0,0 +1,89 @@
+import fs from "fs-extra"
```

Comment:
Extracted to separate module from `packages/gatsby/src/utils/worker/child/render-html.ts`